tox.ini

```diff
@@ -33,7 +33,7 @@ commands =
     pytest {toxinidir}/tests --doctest-plus {posargs}
     pytest {toxinidir}/tests --doctest-plus --doctest-rst {posargs}
     pytest {toxinidir}/tests --doctest-plus --doctest-rst --text-file-format=tex {posargs}
-
+    sphinx-build {toxinidir}/tests {toxinidir}/tests/_build/html -W
```

Comment:
I see that it runs fine, yay! But given we cannot see the rendered doc here, we won't catch the problem where it builds without warning but also doesn't show code snippet... 💭

Reply:
There were a ton of warnings for the empty code snippets, so I'm confident if there is any issues with the directives we will see warnings if not straight our errors. So I would say let's go ahead, and think about moving stuff to circleCI only if any new sphinx related problem comes up.

Reply:
Ok, I thought you said there was no warning on RTD when this happened to astroquery but perhaps I misunderstood.

Anyways, I agree we can defer rendering.

Reply:
Yes, that was my mistake. We were not set up to fail on warnings, and I assumed the "passed" status meant that we are all clean, then locally saw a ton of warnings (not just related to this, but a whole lot of other things crept into the repo in the past many months since we switched CIs...)

Reply:
Ah, okay... 😬